distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc

```diff
@@ -0,0 +1,52 @@
+:title: Validator Plugins on Ingest
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 00
+:summary: Validator plugins run on Ingest
+
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
+depending on whether *Enforce errors* is selected within the admin console. Below is a list
+of the validators run against the data ingested.
+
+[NOTE]
+====
+To navigate to turn on/off *Enforce errors*:
+ . Navigate to the *${admin-console}*.
```

Comment:
Is there an extra space at the start of this line? it could affect your list format.

Reply:
fixed